maestro-orchestra-models/src/main/java/maestro/orchestra/Condition.kt

```diff
@@ -20,6 +21,10 @@ data class Condition(
     }
 
     fun description(): String {
+        if(label != null && label != ""){
```

Comment:
```suggestion
        if(label != null){
```

It's safe to assume that if the user decided to write the `label` that should be the description even if it's empty.